indexer/altitude_loader.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "indexer/feature_altitude.hpp"
+#include "indexer/index.hpp"
+
+#include "3party/succinct/rs_bit_vector.hpp"
+
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+namespace feature
+{
+using TAltitudeSectionVersion = uint16_t;
```

Comment:
Yes. We use it in 3 places. I guess it's ok to use typedef here. But I moved it to AltitudeHeader.